### DISCUSSION TITLE
加密Django代码并设置有效期，到达有效期后程序任在运行，未提示到达有效期

### DISCUSSION THREAD

**Discussion by wei523712** [dashingsoft/pyarmor#1719] [Q&A]
使用pyarmor对django代码进行加密 ，并设置了有效期，当到达有效期那天时，程序任在正常运行，只有退出后再重新启动才会提示到期。
请问有什么方法可以让程序在 到达有效期后 自动退出或者报错退出。

**Comment by jondy** [comment]
请查看参数 `--period` 的用法
